apps/aws-notebook/app.yaml

```diff
@@ -1,49 +1,66 @@
 ---
 overview: |
   # Jupter Notebook
-
+endpoints:
+  - name: jupyter-notebook
+    http:
+      routes:
+        - backend:
+            target: jupyter-notebook
+            port: notebook-service
+          path_prefix: /
 workspaces:
-- name: dev
-  checkouts:
-  - path: demo
-    repo:
-      git: https://github.com/crafting-demo/solutions.git
-    manifest:
-      overlays:
-      - file: apps/aws-notebook/manifest.yaml
-  env:
-  - AWS_CONFIG_FILE=/run/sandbox/fs/secrets/shared/aws-config
-  - OPENVPN_CONFIG_FILE=/run/sandbox/fs/secrets/shared/openvpn-config
-  - ECS_CLUSTER_NAME=
-  - SUBNET_ID=
-  - SERVICE_LAUNCH_TYPE=FARGATE
-  - TASK_IMAGE=
-  - SECURITY_GROUPS=
-  base_snapshot: base-notebook-v1
+  - name: jupyter-notebook
+    ports:
+      - name: notebook-service
+        port: 8888
+        protocol: HTTP/TCP
+    checkouts:
+      - path: dev
+        repo:
+          git: git@github.com:crafting-demo/solutions.git
```

Comment:
This is a public repo, use `https://` otherwise customer won't be able to checkout.

Reply:
done